config/protocol.unit_testnet.single.yml

```diff
@@ -1,7 +1,7 @@
 ProtocolConfiguration:
   Magic: 42
   MaxTraceableBlocks: 200000
-  TimePerBlock: 1s
+  TimePerBlock: 100ms
```

Comment:
Maybe then reduce the block time for test four-nodes chain as far?

Reply:
You're right, it is also used by dump/restore tests and VM CLI tests, but they don't produce blocks anyway, so it's not important.